### ISSUE TITLE
Sorting events on author results in error

### ISSUE THREAD

**Issue by joostclevis** [solspace/craft3-calendar#335]
**Describe the bug or issue you're experiencing**

When sorting the events in the backend on author you get an error:

```
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'author' in 'order clause' The SQL being executed was: SELECT `elements`.`id` FROM (SELECT `elements`.`id` AS `elementsId`, `elements_sites`.`id` AS `siteSettingsId` FROM `elements` `elements` INNER JOIN `calendar_events` `calendar_events` ON `calendar_events`.`id` = `elements`.`id` INNER JOIN `elements_sites` `elements_sites` ON `elements_sites`.`elementId` = `elements`.`id` INNER JOIN `calendar_calendars` ON `calendar_calendars`.`id` = calendar_events.`calendarId` LEFT JOIN `users` ON `users`.`id` = calendar_events.`authorId` WHERE (calendar_events.`calendarId`=2) AND (`elements_sites`.`siteId`=1) AND (`elements`.`archived`=FALSE) AND (`elements`.`dateDeleted` IS NULL) AND (`elements`.`draftId` IS NULL) AND (`elements`.`revisionId` IS NULL) ORDER BY `author`, `calendar_events`.`authorId`, `calendar_events`.`startDate`) `subquery` INNER JOIN `elements` `elements` ON `elements`.`id` = `subquery`.`elementsId` INNER JOIN `elements_sites` `elements_sites` ON `elements_sites`.`id` = `subquery`.`siteSettingsId` INNER JOIN `calendar_events` `calendar_events` ON `calendar_events`.`id` = `subquery`.`elementsId` INNER JOIN `calendar_calendars` ON `calendar_calendars`.`id` = calendar_events.`calendarId` LEFT JOIN `users` ON `users`.`id` = calendar_events.`authorId` ORDER BY `author`, `calendar_events`.`authorId`, `calendar_events`.`startDate`
```

**Steps to reproduce**

1. Go to calendar
2. Click on events
3. Sort on author

**Expected behavior**
Sort the events on author and don't give an error message

**Craft & Plugin Info (please complete the following information):**

- Craft Version:  5.4.7.1 
- Calendar Version: 5.0.10
- Calendar Edition: Lite
- Fresh Install or Upgrade: Upgrade

**Additional context**

**Comment by kjmartens** [comment]
Thanks for reporting this @joostclevis.

We'll check into it shortly. 🙂

**Comment by kjmartens** [comment]
This is now fixed in Calendar 5.0.11.
